Cargo.toml

```diff
@@ -44,6 +44,7 @@ log = "0.3"
 env_logger = "0.4"
 getopts = "0.2"
 derive-new = "0.5"
+time = "0.1"
```

Comment:
Could you use std::time instead of the time crate? I think it has everything you need without the extra dep.